tests/test_commands.py

```diff
@@ -766,3 +766,272 @@ def test_cmlist(mapdl):
     assert len(cmlist_all.to_array()) == len(cmlist_all.to_list())
     for each_ in cmlist_all.to_list():
         assert each_ in cmlist_all
+
+
+class Test_bc_cmdlist_solid:
+
+    def solid_model(self, mapdl):
+        # Solid model (Geometry)
+
+        mapdl.clear()
+
+        mapdl.prep7()
+
+        # Define keypoints, lines and area
+        # --------------------
+        mapdl.k(1, 0, 0)
+        mapdl.k(2, 1, 0)
+        mapdl.k(3, 1, 1)
+        mapdl.l(1, 2)
+        mapdl.l(2, 3)
+        mapdl.l(3, 1)
+        mapdl.a(1, 2, 3)
+
+        # Define a material
+        # --------------------
+        mapdl.mp("EX", 1, 30e6)
+        mapdl.mp("NUXY", 1, 0.25)  # Poisson's Ratio
+
+        # Define section
+        # --------------------
+        mapdl.et(1, "PLANE183")
+        mapdl.keyopt(1, 1, 0)
+        mapdl.keyopt(1, 3, 3)
+        mapdl.keyopt(1, 6, 0)
+        mapdl.r(1, 0.01)
+
+    def test_dklist(self, mapdl):
+
+        df_dk = pd.DataFrame(
```

Comment:
Tests are failing because the ``import pandas as pd`` doesn't appear at the top of this file. Could you add it whenever you get the chance please?

Reply:
Camille, thanks for your comment. Now it is done.

Reply:
Thank you @pvargasm. Let me recheck it and I'll approve it!